lib/write-aggregated-translations.js

```diff
@@ -3,7 +3,7 @@ const chalk = require('chalk');
 const supportedLocales = require('../config/i18nSupportedLocales');
 const generateTranslationFile = require('./generate-translation-file');
 
-const writeAggregatedTranslations = (aggregatedMessages, locales, fileSystem, outputDir) => {
+const writeAggregatedTranslations = (aggregatedMessages, locales, fileSystem, outputDir, isReactIntlV5) => {
```

Comment:
I'm being pedantic here, but technically the lack of intl shape indicates that it's not `react-intl` v2, not that it is `react-intl` v5 since it was removed in v3.
![](https://media.giphy.com/media/13aSSyJaI5NkTm/giphy.gif)

Reply:
True. I went back and forth about that a little bit, but wasn't thinking we were really wanting to support v3 or v4